spec/src/main/asciidoc/core/beanmanager_lite.asciidoc

```diff
@@ -0,0 +1,268 @@
+[[programmatic_access]]
+
+== Programmatic access to container
+
+CDI 3.0 used to provide a single `BeanManager` object, which allows access to many useful operations.
+In CDI 4.0, this is split into `BeanManagerLite` and `BeanManager`.
+
+`BeanManagerLite` provides access to a subset of `BeanManager` features which can be implemented in more restricted environments;
+It is available in {cdi_lite} environments.
+
+`BeanManager` extends `BeanManagerLite` and provides the remaining features.
+It is available in {cdi_full} environments.
```

Comment:
{cdi_full} environments environments -> a {cdi_full} environment

Reply:
The plural form here is correct because there are two CDI Full environments - SE and EE

Reply:
I'll try to rephrase this paragraph today, there are too many comments so I clearly have to do a better job :-)